src/index.js

```diff
@@ -140,6 +140,18 @@ class ServerlessAppsyncPlugin {
     };
   }
 
+  getDeltaSyncConfig(config) {
+    if (config && config.deltaSyncConfig) {
+      return {
+        BaseTableTTL: config.deltaSyncConfig.baseTableTTL,
+        DeltaSyncTableName: config.deltaSyncConfig.deltaSyncTableName,
+        DeltaSyncTableTTL: config.deltaSyncConfig.deltaSyncTableTTL,
```

Comment:
@yyoshiki41 what would happen if one of `baseTableTTL`, `deltaSyncTableName` or `deltaSyncTableTTL` is undefined?

Does AppSync use defaults?

Reply:
@bboure 

No, AppSync does not have default values.
error occurred when one of these fields is undefined.
### CloudFormation Error
```
2020-03-14 19:00:27 UTC+0900
GraphQlDsDelta
UPDATE_FAILED
Property validation failure:
  [The property {/DynamoDBConfig/DeltaSyncConfig/BaseTableTTL} is required]
```

We need to set all of params `baseTableTTL`, `deltaSyncTableName` and `deltaSyncTableTTL`, if `versioned` is true.

Reply:
Would it make sense to add some defaults ourselves?
At least for TTL values? Not sure what would be a good value.

Additionally, we could throw an error when tableName or TTL values (if we don't use defaults) are undefined, before it gets to Cfn.

Reply:
OK, I'll prepare TTL default values.
(There is no value that can cover all cases, but I think it is not harmful.)

- baseTableTTL: 0
The amount of time (in minutes) items should be kept in the base table when deleted.
Set to 0 to delete items in the base table immediately.
- deltaSyncTableTTL: 60
DeltaSync table will keep track of changes in 60 mins.
_NOTE:_ With a high velocity of data, it may make sense to keep this short. Alternatively, if your clients are offline for longer periods of time, it might be prudent to keep this longer.

> Additionally, we could throw an error when tableName or TTL values (if we don't use defaults) are undefined, before it gets to Cfn.

I'll implements the validation for `deltaSyncConfig` params.

Thanks for your review!

Reply:
set default TTL values 
https://github.com/sid88in/serverless-appsync-plugin/pull/313/commits/243527d7ec453844da9eb71b706e50aa2f03ca3e